ibm/service/power/data_source_ibm_pi_cloud_connection.go

```diff
@@ -7,129 +7,127 @@ import (
 	"context"
 	"log"
 
-	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
-
 	"github.com/IBM-Cloud/power-go-client/clients/instance"
-	"github.com/IBM-Cloud/power-go-client/helpers"
 	"github.com/IBM-Cloud/power-go-client/power/models"
 	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 )
 
 const (
 	PICloudConnectionId               = "cloud_connection_id"
-	PICloudConnectionName             = "name"
-	PICloudConnectionSpeed            = "speed"
-	PICloudConnectionGlobalRouting    = "global_routing"
-	PICloudConnectionMetered          = "metered"
 	PICloudConnectionStatus           = "status"
-	PICloudConnectionClassicEnabled   = "classic_enabled"
-	PICloudConnectionUserIPAddress    = "user_ip_address"
 	PICloudConnectionIBMIPAddress     = "ibm_ip_address"
+	PICloudConnectionUserIPAddress    = "user_ip_address"
 	PICloudConnectionPort             = "port"
-	PICloudConnectionNetworks         = "networks"
-	PICloudConnectionClassicGreDest   = "gre_destination_address"
 	PICloudConnectionClassicGreSource = "gre_source_address"
-	PICloudConnectionVPCEnabled       = "vpc_enabled"
-	PICloudConnectionVPCCRNs          = "vpc_crns"
 	PICloudConnectionConnectionMode   = "connection_mode"
 )
```

Comment:
The constants here were also used in the resource files. Eventually, I want the constants defined here gone and constants only defined in ibm_pi_constants.go. For now, to avoid having to change the data source files multiple times and also avoid compilation issues with resource files move these to ibm_pi_constants.go.